doc/rados/operations/erasure-code-jerasure.rst

```diff
@@ -7,8 +7,8 @@ also the default for Ceph erasure coded pools.
 
 The *jerasure* plugin encapsulates the `Jerasure
 <https://github.com/ceph/jerasure>`_ library. It is
-recommended to read the ``jerasuredocumentation to get a better
-understanding of the parameters. Note that the ``jerasure.org``
+recommended to read the ``jerasur`` edocumentation to
```

Comment:
``jerasure`` documentation

Reply:
Feh, single-parenting this week.  pushed.